content/en/docs/reference/kubectl/docker-cli-to-kubectl.md

```diff
@@ -36,10 +36,18 @@ kubectl:
 
 ```shell
 # start the pod running nginx
-kubectl run --image=nginx nginx-app --port=80 --env="DOMAIN=cluster"
+kubectl create deployment --image=nginx nginx-app
```

Comment:
@Cweiping - what port is the container in new Deployment listening on?

Reply:
Does `kubectl expose` work if you didn't specify a port there?

Reply:
According to my test results, deployment can also be accessed using pod IP and port without specific port. If `kubectl expose` is used, the unified endpoint can be directly used to route to multiple pods.

Reply:
Cool